activerecord/lib/active_record/connection_adapters/trilogy_adapter.rb

```diff
@@ -76,6 +76,10 @@ def initialize_type_map(m)
       def initialize(config, *)
         config = config.dup
 
+        # Trilogy ignore `socket` if `host is set. We want the opposite to allow
```

Comment:
```suggestion
        # Trilogy ignores `socket` if `host` is set. We want the opposite to allow
```

Reply:
This was fixed in ee65e97829e1d09a00047958273656d3b6488924 on main, if someone wants to backport it :bow: /cc @byroot